README.md

```diff
@@ -41,6 +41,15 @@ limitations under the License.
 - Open `.env` files and set `REACT_APP_API_URL` to point to services:        
     - local: [http://localhost:{PORT}](http://localhost:{PORT})      
 
+<h3>💻 UI Customization</h3>
+
+The default description on each of the three main application pages can be customized by setting the following environment variables:
+
+- `DESCRIPTION_RETRIEVAL`
```

Comment:
If I am not mistaken, we agreed on exposing description for each collection via flowing that information through APIs rather than environment variables? 

Did we decide to not experiment with that approach?